core-bundle/tests/Functional/RoutingTest.php

```diff
@@ -770,8 +770,8 @@ public function getRootAliases(): array
             ],
             'Throws a "no root page found" exception if no language matches' => [
```

Comment:
Renders the 404 page…

Reply:
Fixed in 5496681b13cec9843806184597d48623796eab1c